### ISSUE TITLE
nmrCV

### ISSUE THREAD

**Issue by proccaserra** [OBOFoundry/OBOFoundry.github.io#1162]
Use this form to register a new ontology with the OBO Foundry. Please read the instructions provided here:
http://obofoundry.org/docs/NewOntologyRegistrationInstructions.html

## Ontology title
nmrCV

## Requested ID space
nmrcv

## Contact person
Name: Philippe Rocca-Serra
Email address: proccaserra@gmail.com
GitHub username: proccaserra

proxy for the original authors:

Name:  Daniel Jacob
email address: daniel.jacob@u-bordeaux.fr
github username: djacob65

Name: Daniel Schober:
email address: Daniel.Schober@ipb-halle.de
github username: DSchober

## Issue tracker
https://github.com/MSI-Metabolomics-Standards-Initiative/nmrcv/issues

## Ontology license

-  [CC-BY version 4](https://creativecommons.org/licenses/by/4.0/)

## Available ontology formats
OWL RDF
(OBO)

## What domain is the ontology intended to cover?
1D & 2D NMR spectroscopy 

## Related OBO Foundry ontologies
BFO as top level ontology

inclusion in the [Metabolomics Society Initiative Ontology](https://github.com/MSI-Metabolomics-Standards-Initiative/MSIO), an application ontology which uses:

BFO
IAO
OBI as mid level ontology
CHEBI for all chemical entities (imports tested but not in production yet)
CHMO for NMR acquisition techniques (e.g. pulse sequences)  (imports tested but not in production yet)
UO for all units
PS-MS

## Intended use/related projects
supporting [nmrML format](http://nmrml.org/)

## Additional comments or remarks
As per discussion with @cmungall

**Comment by cmungall** [comment]
It has been a month with no objections, this should be added to OBO

I have made some OBO-esque comments on your tracker, acceptance is not contingent on addressing these:
https://github.com/MSI-Metabolomics-Standards-Initiative/nmrcv/issues/2

**Comment by cmungall** [comment]
Is nmrCV really your ontology title? I recommend a non-abbreviation.

@proccaserra do you know how to make PRs to get your ontology metadata in or do you need help with this?

**Comment by proccaserra** [comment]
@cmungall thx for getting back to us. 'Nuclear Magnetic Resonance Ontology' could be the title to mimic the PSI-MS vocabulary, with which nmrCV shares a lot when it comes to its structure (little axiomatisation, limited class metadata in its current form. 
Regarding the PR, yes I am aware of the process and its associated documentation (or I could test the nifty metadata editor @jamesaoverton produced.

**Comment by nlharris** [comment]
What is the status of this?

**Comment by proccaserra** [comment]
Hi @nlharris I am overdue a PR , my bad

**Comment by nlharris** [comment]
no worries, @proccaserra, these are crazy times! i just wanted to make sure you weren't waiting for something from us.

**Comment by nlharris** [comment]
Should we still leave this open?

**Comment by pfabry** [comment]
Hi @proccaserra,
Since there has been no activity on this issue for quite a long time, we assume that it is no longer relevant and choose to close it to facilitate issues handling on this repository. Please let me know if there are any questions about this process and feel free to reopen it if you think it is still of interest.
